src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonInputFormatter.cs

```diff
@@ -104,6 +105,8 @@ public class JsonInputFormatter : TextInputFormatter
             SupportedMediaTypes.Add(MediaTypeHeaderValues.ApplicationAnyJsonSyntax);
         }
 
+        public virtual bool SendBadRequestForExceptionsDuringDeserialization => false;
```

Comment:
I think we should make this property intialized with `GetType() == typeof(JsonInputFormatter). Then anyone who's subclassed it sees no change.